third-party/realdds/src/dds-participant.cpp

```diff
@@ -133,6 +134,11 @@ void dds_participant::init( dds_domain_id domain_id, std::string const & partici
     // Indicates for how much time should a remote DomainParticipant consider the local DomainParticipant to be alive.
     pqos.wire_protocol().builtin.discovery_config.leaseDuration = { 10, 0 };  // [sec,nsec]
 
+    //Disable shared memory, use only UDP
+    auto udp_transport = std::make_shared<eprosima::fastdds::rtps::UDPv4TransportDescriptor>();
+    pqos.transport().use_builtin_transports = false;
```

Comment:
This is good code to have, but only optional. See my other message. I'd even consider making this a CMake option when `BUILD_WITH_DDS` is on.

Reply:
See Nir's comment. Will leave for this PR, consider changing to parameter in next PRs